HackYourFuture-CPH/FP-annotatetheweb-class10
# #195 Error messages

Changes in this PR:

- On project page, if there's an error/timeout, we get an error message where user can navigate back to homepage to try again, or reload the page

- On LoginPage: if user does not fill the input fields, an alert pops up. And if there's an error from firebase, an error pops up with corresponding message. 

- On register page: error messages (in alert) should appear if user do not fill all fields, uses wrong format for email, email is already used

- added charset for database, so users can have special characters in their names

Testing: 
- Project page: wait for the timeout, the error message will appear, try the buttons. 
- Project page: use wrong url for screenshot on homepage. Then You should get the same error message.

LoginPage: 
- do not fill any input fields
- fill only one input field
- use wrong username, use wrong password

Register Page:
- do not fill any fields
- fill only few of the fields
- use incorrect email (like not including '@' for example
- use email that is already used
